Dockerfile

```diff
@@ -186,3 +186,9 @@ COPY --from=releaser /out/ /
 # see open-pr job in .github/workflows/docs.yml for more details
 FROM scratch AS docs-reference
 COPY docs/reference/*.yaml .
+
+FROM base AS release-image
```

Comment:
The base image is go/alpine, with docker and git installed in it. Anything else that we think should go in here?

Reply:
I notice we're installing docker from alpine's apk repositories, which are builds of docker built by alpine. Perhaps that's ok for testing, but we definitely should _not_ ship those as part of images for end-users.

Having a "runnable" compose image with docker installed basically replicates the official `docker:cli` image; see https://github.com/docker-library/docker/pull/369

Reply:
Good point, I think basing compose off the official docker cli image would be a lot better.

Reply:
Oh, nevermind! Compose is in the latest docker:*-cli images. Then I'll see if we can use that.